src/Http/TestResponse.php

```diff
@@ -117,6 +117,16 @@ public function assertUnprocessable(): self
         return $this->assertStatus(422);
     }
 
+    public function assertBodyContains(string $needle): self
+    {
+        TestCase::assertStringContainsString(
+            $needle,
+            (string) $this->response->getBody()
+        );
```

Comment:
Could you change default error message? For example `Response doesn't contain [%s]` ?